.github/ci-scripts/bash/functions-for-build

```diff
@@ -565,6 +575,64 @@ setup_qa_env() {
   deactivate
 }
 
+setup_packer_env() {
+  sudo python3 -m pip uninstall -y packer_framework
+  python3 -m pip install --user virtualenv --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m venv pkenv
+  source pkenv/bin/activate
+  python3 -m pip install wheel --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m pip install -r ./requirements.txt --extra-index-url=https://$GITHUB_USER:$GITHUB_TOKEN@${NEXUS_SERVER_ENDPOINT}/repository/${NEXUS_PULL_REPOS_PY}/simple
+
+  find /usr/bin /usr/local/bin -name build_all_artifacts
```

Comment:
What is this for?

Reply:
to check if the framework is installed or not

Reply:
How does it help. `find` does not fail if it won't find the file